src/main/java/org/carlmontrobotics/commands/ArmTeleop.java

```diff
@@ -28,6 +47,7 @@ public void initialize() {}
   @Override
   public void execute() {
     double value = joystick.getAsDouble();
+    // use trapazoid math and controllerMoveArm method from arm subsytem to apply voltage to the motor
```

Comment:
is controllerMoveArm() in the arm class a suitable substitute for doing this manually?
If so, do you need to be using a fully-fledged command?
(Options: InstantCommand, CommandGroup, this)

Reply:
fix this~